server/lib/users/user-api.ts

```diff
@@ -139,6 +142,12 @@ const relationshipsThrottle = createThrottle('accountrelationships', {
   window: 60000,
 })
 
+const changeLanguageThrottle = createThrottle('changelanguage', {
```

Comment:
I think it is more expensive to check this throttle than it is to change a single DB column so I don't know that this is really worth it